ReactWindows/ReactNative.Shared/Reflection/EnumHelpers.cs

```diff
@@ -41,6 +39,30 @@ public static T Parse<T>(string value)
             return Parse<T>(value);
         }
 
+        private static Dictionary<string, object> EnumToDictionary(Type type)
+        {
+            var result = new Dictionary<string, object>();
+
+            var names = Enum.GetNames(type);
+            var values = Enum.GetValues(type);
+
+            for (int i = 0; i < values.Length; ++i)
+            {
+                string name = names[i];
+                object value = values.GetValue(i);
+
+                result.Add(Normalize(name), value);
+
+                var enumMemberAttribute = type.GetField(name).GetCustomAttributes(typeof(EnumMemberAttribute), true).SingleOrDefault();
+                if (enumMemberAttribute != null)
+                {
+                    result.Add(Normalize(((EnumMemberAttribute)enumMemberAttribute).Value), value);
```

Comment:
>EnumMemberAttribute [](start = 43, length = 19)

Just curious, does the serialization behavior of `EnumMemberAttribute` only work in one direction? I.e., is the value only used to serialize to string, or is there some tool that can be used for parsing as well?